docs/100-upgrade/migration-guides/3.2-3.3.mdx

```diff
@@ -120,3 +152,58 @@ only work when creating response with `json()` without passing a second
 parameter.
 
 :::
+
+### Deprecate `process.env.FRONT_COMMERCE_WEB_*` usage
+
+**Breaking:** In this release, we updated the way we handle public
```

Comment:
**thought:** Should we add a "breaking" badge component to make those case even easier to spot?

Reply:
![BreakingBadChemistryGIF](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/712d8cf8-6b32-4fce-9506-0eaadef75df6)


It would be good, but I don't have capacity to add it at the moment.